scrapers/errata.py

```diff
@@ -187,7 +186,7 @@ def get_associated_builds(self, advisory_id):
             LEFT JOIN Errata_public.brew_builds AS brew_builds
                 ON brew_builds.id = brew_mappings.brew_build_id
             LEFT JOIN Errata_public.packages AS packages
-                ON brew_builds.package_id = packages.id WHERE errata_id = {0};
+                ON brew_builds.package_id = packages.id WHERE errata_id = {0}
```

Comment:
This change isn't necessary, but if you want to do it, could you do it for all the scrapers and all the queries? 

Or could you add the `;` to other places where it is missing?